doc/manuals/user/update_operators.md

```diff
@@ -69,388 +69,4 @@ Thus, update operators solve both problems:
 * Complexity. Now applications do not need to do a read-calculate-update cycle. Just updating
   (with the operator) is enough.
 * Race conditions. This is solved due to Orion ensures that these operations are executed atomically
-  on the context attributes. Many applications can update the same piece of context without problems.
-
-## Supported operators
```

Comment:
After the simplification in this document, the `## Introduction` is not needed any more

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4182/commits/1c35ef65f545d3efcfbf2bbee6108d311c2f6c04